tests/python/test_templates_prh.py

```
@@ -35,3 +44,88 @@ def test_javascript_rule_uses_word_boundary_regex_for_js() -> None:
+
+    prh は同一 rule 内の複数 pattern を内部で alternation に合成し /g 適用するため，
+    `[/ X/, /X /]` のように pattern を分けても両側スペース（例: `CI ・ cron`）の
+    後続スペースが消えずに spec test が落ちる．長い順 alternation `/ X | X|X /` で
+    leftmost-longest を機能させ，両側スペースを 1 マッチで `X` に置換する設計．
+    """
+    rule = _find_rule_by_expected(prh_rules, symbol)
+    assert rule is not None
+    patterns = rule.get("patterns") or []
+    expected_regex = f"/ {symbol} | {symbol}|{symbol} /"
+    assert expected_regex in patterns, (
+        f"templates/prh.yml の expected:{symbol} rule に `{expected_regex}` パターンが無い．"
```

Ruff (0.15.12)

[warning] String contains ambiguous `．` (FULLWIDTH FULL STOP). Did you mean `.` (FULL STOP)?

(RUF001)